misc/sync_scripts.py

```diff
@@ -0,0 +1,48 @@
+import os
+import ruamel.yaml
+
+# Scripts located in this project
+PRE_CONVERSION_SCRIPT_PATH = "scripts/preconversion_assessment_script.py"
+CONVERSION_SCRIPT_PATH = "scripts/conversion_script.py"
+
+# Yaml playbooks in rhc-worker-script
+PRE_CONVERSION_YAML_PATH = os.path.join(
+    "..", "rhc-worker-script/development/nginx/data/convert2rhel_analysis.yml"
+)
+CONVERSION_YAML_PATH = os.path.join(
+    "..", "rhc-worker-script/development/nginx/data/convert2rhel_conversion.yml"
+)
```

Comment:
Maybe these could be just the path inside the rhc-worker-script folder and the path to the worker repo itself could be variable in makefile so everyone can adjust it based on their structure.

then the make command would just take the path to the repo and pass it to the script?

Something like:

```suggestion
PRE_CONVERSION_YAML_PATH = os.path.join(
    sys.argv[1], "development/nginx/data/convert2rhel_analysis.yml"
)
CONVERSION_YAML_PATH = os.path.join(
    sys.argv[1], "/development/nginx/data/convert2rhel_conversion.yml"
)
```

Reply:
We can keep it simple right now, the sync is in separate epic and will be solved in future.